workers/shared.py

```diff
@@ -86,20 +87,32 @@ def hdfs_to_local_path(hdfs_path):
 
     return filename
 
+def guess_ensembl_release(filepath):
+    try:
+        release = varcode.load_vcf(filepath)[0].ensembl.release
```

Comment:
unfortunately yes. We have two other alternatives to this one:
1) Copy the logic within varcode over to Cycledash and pass the `vcf_header` field through that logic
2) Convince @iskandr to expose relevant guessing methods in https://github.com/hammerlab/varcode/blob/master/varcode/reference.py and use them directly